pymeasure/instruments/keithley/keithley6221.py

```diff
@@ -93,6 +93,16 @@ def __init__(self, adapter, name="Keithley 6221 SourceMeter", **kwargs):
         map_values=True
     )
 
+    shield_connection = Instrument.control(
+        ":OUTPut:ISHield?", ":OUTPut:ISHield %s",
+        """ A string property that controls
+        to whom the shield is connected, default to 'output-low'.
+        Valid values are 'guard' and 'output-low'. """,
+        validator=strict_discrete_set,
+        values={"guard": "GUARd", "output-low": "OLOW"},
+        map_values=True
+    )
```

Comment:
If it is just a two-way-option, a boolean property might be better.

For example `shield_to_guard_enabled = True/False` will be easy to understand and use.